### ISSUE TITLE
Expanding properties changes object order

### ISSUE THREAD

**Issue by geekus** [Turbasen/Turbasen#99]
When expanding a property of an object, like `bilder` to get their properties, the order of the related objects are not then same as the `bilder` list of ids when not expanding.

Test: https://dev.nasjonalturbase.no/steder/57b18381cc7f890001d3a121 vs. https://dev.nasjonalturbase.no/steder/57b18381cc7f890001d3a121?expand=bilder

**Comment by hhartz** [comment]
@geekus yes I see the same issue here - causing inconsistent references to foreground and background images
